src/StudentDashboard/QuestionArea.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import './QuestionArea.scss'
+import { incrementScore, incrementCurrentQuestion, setLessonOver } from '../actions/index';
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+
+function QuestionArea(props) {
```

Comment:
We have been destructuring the props for readability but this is not a big deal.  We never talked about this in our standups.

Reply:
I can be sure to do that from here on out